src/controllers/authentication.controller.js

```diff
@@ -32,6 +34,8 @@ export default class AuthenticationController {
       const saveUser = await handleSignUp(userData);
       const savedUserObject = _.omit(saveUser, 'password');
       const token = await generateToken(savedUserObject);
+      console.log(req.body.email);
+      await sendEmail.sendSignUpVerificationLink(req.body.email, `${process.env.APP_URL}`);
       return successResponse(res, statusCodes.created, customMessages.userSignupSuccess, token);
```

Comment:
As you send the email after successful registration, don't think that you should edit the message returned on this signup API? Because after a user has successfully created the account see on the screenshot the mockup of the page where he/she will be redirected 
![Screenshot from 2020-03-11 14-30-32](https://user-images.githubusercontent.com/56886064/76417109-ec643d00-63a4-11ea-8703-7eb456549852.png)
that is the page where a user will be redirected after successful registration, but as I see you are still sending the token key and the message on the API 
![Screenshot from 2020-03-11 14-32-17](https://user-images.githubusercontent.com/56886064/76417205-21708f80-63a5-11ea-87ca-c8562a47c1e9.png)
that is what I got after trying to signup I suggest you change this signup message a user receives after successful signup. It should match what is on the mockup.